misc/build.gradle.kts

```diff
@@ -46,7 +46,7 @@ android {
 
 }
 dependencies {
-    val composeBom = platform(libs.androidx.compose.bom)
+ val composeBom = platform(libs.androidx.compose.bom)
```

Comment:
Nit - this formatting change is unnecessary

Reply:
Restored. One of the formatters did that.--I think Android Studio.